### PR TITLE
[docs] Add sso documentation

### PR DESCRIPTION
**Pull request by sjwiesman**
### Motivation

Add basic documentation for saml and oidc based sso in Materialize console.

### CODE REVIEW

File: doc/user/content/manage/sso.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: "Single Sign-On"
+description: "Configure single sign-on using SAML or Open Id Connect"
+menu:
+  main:
+    parent: manage
+    weight: 10
+---
+
+Materialize allows for single sign-on (SSO) using your existing [SAML](https://auth0.com/blog/how-saml-authentication-works/) or [Open Id Connect](https://auth0.com/intro-to-iam/what-is-openid-connect-oidc) based identity provider.
+With SSO, users can log in to the Materialize console using the same ID and password as they do for other systems.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:11]:
```suggestion
You can configure single sign-on (SSO) as an additional layer of account security using your existing [SAML](https://auth0.com/blog/how-saml-authentication-works/)- or [OpenID Connect](https://auth0.com/intro-to-iam/what-is-openid-connect-oidc)-based identity provider. This ensures that all users can securely log in to the Materialize console using the same authentication scheme and credentials across all systems in your organization.
```

File: doc/user/content/manage/sso.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Materialize allows for single sign-on (SSO) using your existing [SAML](https://auth0.com/blog/how-saml-authentication-works/) or [Open Id Connect](https://auth0.com/intro-to-iam/what-is-openid-connect-oidc) based identity provider.
+With SSO, users can log in to the Materialize console using the same ID and password as they do for other systems. 
+
+{{< note >}}
+Single sign-on in Materialize only supports authentication into the Materialize console. Permissions within the database are handled separately using [role-based access control](/manage/access-control/).
+{{</ note >}}
+
+## Prerequisites
+
+* You must have an existing SAML or Open Id Connect based identity provider.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:19]:
```suggestion
* You must have an existing SAML- or OpenID Connect-based identity provider.
```

File: doc/user/content/manage/sso.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* Only users assigned the `OrganizationAdmin` role can view and modify SSO settings.
+
+## SAML
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:22]:
```suggestion
## Configure SAML authentication
```

File: doc/user/content/manage/sso.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+1. Select *Organization Admin* or *Organization Member* depending on what level of console access the user needs:
+
+    - **Organization Admin**: Can invite new users, edit account information,
+    and edit account security information.
+    - **Organization Member**: Can login to the console.
+
+    These roles do not refer to RBAC roles and privileges and refer to
+    actions users can take in the Materialize Console.
+
+## Open Id Connect
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:43]:
```suggestion
## Configure OpenID Connect authentication
```

File: doc/user/content/manage/sso.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+{{</ note >}}
+
+## Prerequisites
+
+* You must have an existing SAML or Open Id Connect based identity provider.
+* Only users assigned the `OrganizationAdmin` role can view and modify SSO settings.
+
+## SAML
+
+1. [Login to the Materialize console](https://console.materialize.com/).
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:24]:
```suggestion
1. [Log in to the Materialize console](https://console.materialize.com/).
```

File: doc/user/content/manage/sso.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Navigate to Account > Account Settings > SSO.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:26]:
```suggestion
1. Navigate to **Account** > **Account Settings** > **SSO**.
```

File: doc/user/content/manage/sso.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Add the SSO endpoint and public certificate provided by your identify provider. Click proceed.
+
+1. Optionally, add the SSO domain provided by your identify provider. Click proceed.
+
+1. Select *Organization Admin* or *Organization Member* depending on what level of console access the user needs:
+
+    - **Organization Admin**: Can invite new users, edit account information,
+    and edit account security information.
+    - **Organization Member**: Can login to the console.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:38]:
```suggestion
    - **Organization Member**: Can log in to the console.
```

File: doc/user/content/manage/sso.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+    These roles do not refer to RBAC roles and privileges and refer to
+    actions users can take in the Materialize Console.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:41]:
```suggestion
    It's important to note that these roles determine which actions users can take
    in the Materialize console, and do not translate to RBAC roles and privileges
    in the database.
```

Comment (morsapaes) [reply doc/user/content/manage/sso.md:41]: `OrganizationAdmins` are _superusers_ in the database, though, no?

File: doc/user/content/manage/sso.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+## Open Id Connect
+
+1. [Login to the Materialize console](https://console.materialize.com/).
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:45]:
```suggestion
1. [Log in to the Materialize console](https://console.materialize.com/).
```

File: doc/user/content/manage/sso.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Navigate to Account > Account Settings > SSO.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:47]:
```suggestion
1. Navigate to **Account** > **Account Settings** > **SSO**.
```

File: doc/user/content/manage/sso.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Click **Add New** and choose the `Open ID Connect` connection type. 
+
+1. Add your issuer url, client id, and secret key provided by your identify provider. Click proceed.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:51]:
```suggestion
1. Add the issuer URL, client ID, and secret key provided by your identity provider.
```

File: doc/user/content/manage/sso.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Optionally, add the SSO domain provided by your identify provider. Click proceed.
```
Comment (morsapaes) [inline doc/user/content/manage/sso.md:53]:
```suggestion
1. Optionally, add the SSO domain provided by your identity provider. Click **Proceed**.
```

**Review by morsapaes — CHANGES_REQUESTED** [review]
Thanks for helping out, @sjwiesman! The instructions are repeated, so we should probably just slap the SAML- and OpenID connect-specific bits into tabs. If you give it a pass, I can push that in and also create a dedicated "Access and security" section for this page + "Invite users"; it feels awkward to mix it up with dbt and Terraform.

File: doc/user/content/manage/sso.md (hunk 8 above)
Comment (sjwiesman) [reply doc/user/content/manage/sso.md:41]: You're right

**Comment by sjwiesman** [comment]
@morsapaes done

**Pull request merged**